doc/menu_lua_api.txt

```diff
@@ -1,5 +1,5 @@
 Minetest Lua Mainmenu API Reference 0.5.0
```

Comment:
5.0.0

Reply:
Will be solved by rebasing. This branch is simply outdated.

Reply:
Done, thanks for pointing that out.